Documentation/ceph-upgrade.md

```diff
@@ -51,7 +51,7 @@ With this upgrade guide, there are a few notes to consider:
 
 Unless otherwise noted due to extenuating requirements, upgrades from one patch release of Rook to
 another are as simple as updating the common resources and the image of the Rook operator. For example, when Rook v1.5.1 is
-released, the process of updating from v1.5.0 is as simple as running the following:
+released, the process of updating from v1.5.1 is as simple as running the following:
```

Comment:
```suggestion
released, the process of updating from v1.5.0 to v1.5.1 is as simple as running the following:
```

Reply:
actually I'll just revert that change, the full sentence is already talking about updating to v1.5.1.